src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -422,9 +422,28 @@
     },
     "http": {
       "displayName": "HTTP",
-      "description": "Register native handlers for http and setup Refit support",
+      "description": "Register native handlers for http.",
```

Comment:
Let's make sure we're consistent with the other descriptions, see if they end with a period or not.

Reply:
This should now reflect that this is a choice, like the other choice parameters. So it should be something like:

"Configures http support in the application"